frontend/package.json

```diff
@@ -125,17 +125,20 @@
     "serve": "next start",
     "test": "playwright test",
     "e2e-save": "playwright test",
-    "e2e": "playwright test --project chromium --grep-invert @loggedIn",
+    "e2e": "playwright test --project chromium",
     "e2e-pr": "playwright test --project $PROJECT --shard $SHARD --grep-invert @loggedIn",
     "e2e:chrome": "playwright test --project chromium",
     "e2e:chrome:headed": "playwright test --headed --project chromium",
     "e2e:ff": "playwright test --project firefox",
     "e2e:ie": "playwright test --project edge",
-    "e2e-dev": "TEST_ENV=dev playwright test --project chromium --grep-invert @loggedIn",
+    "e2e-dev": "SKIP_LOGIN=true TEST_ENV=dev playwright test --project $PROJECT --shard $SHARD --grep-invert @loggedIn",
+    "e2e-dev-logged-in": "TEST_ENV=dev playwright test --project chromium --grep @loggedIn",
     "e2e-dev-save": "TEST_ENV=dev playwright test",
-    "e2e-staging": "TEST_ENV=staging playwright test --project chromium --grep-invert @loggedIn",
+    "e2e-staging": "SKIP_LOGIN=true TEST_ENV=staging playwright test --project $PROJECT --shard $SHARD --grep-invert @loggedIn",
+    "e2e-staging-logged-in": "TEST_ENV=staging playwright test --project chromium --grep @loggedIn",
     "e2e-staging-save": "TEST_ENV=staging playwright test",
-    "e2e-prod": "TEST_ENV=prod playwright test --project chromium --grep-invert @loggedIn",
+    "e2e-prod": "SKIP_LOGIN=true TEST_ENV=prod playwright test --project $PROJECT --shard $SHARD --grep-invert @loggedIn",
+    "e2e-prod-logged-in": "TEST_ENV=prod playwright test --project chromium --grep @loggedIn",
```

Comment:
update commands to use shards and create logged in specific e2e test command per deployed environment